docs/Contributing/API-for-contributors.md

```diff
@@ -765,6 +766,34 @@ If the credentials are valid, the server redirects the client to the Fleet UI. T
 - `profile_token` is a token that can be used to download an enrollment profile (.mobileconfig).
 - `eula_token` (optional) if an EULA was uploaded, this contains a token that can be used to view the EULA document.
 
+### Over the air enrollment
+
+This endpoint handles over the air (OTA) MDM enrollments
+
+`POST /api/v1/fleet/mdm/ota`
```

Comment:
```suggestion
`POST /api/v1/fleet/ota_enrollment`
```

Reply:
sgtm! feel free to commit the suggestion

Reply:
Recently we stopped using `/mdm` in contributor API for new endpoints, so if we decide to move to public docs we don't need to change it.